test/Microsoft.ML.CodeGenerator.Tests/ApprovalTests/ConsoleCodeGeneratorTests.cs

```diff
@@ -41,7 +41,8 @@ public void ConsoleAppModelBuilderCSFileContentOvaTest()
                 LabelName = "Label",
                 ModelPath = "x:\\models\\model.zip"
             });
-            var result = consoleCodeGen.GenerateConsoleAppProjectContents(namespaceValue, typeof(float), true, true, false);
+            var result = consoleCodeGen.GenerateConsoleAppProjectContents(namespaceValue, typeof(float), true, true,
```

Comment:
All these booleans make it super hard to read what is happening here. I suggest to use named parameters to make it easy to read:

` consoleCodeGen.GenerateConsoleAppProjectContents(namespaceValue, typeof(float), includeLightGbmPackage: true, includeMklComponentsPackage: true, ...`

Reply:
this is a test file and this is how it was before as well.

---
In reply to: [339651958](https://github.com/dotnet/machinelearning/pull/4395#discussion_r339651958) [](ancestors = 339651958)

Reply:
You are adding to the problem here. When making an existing problem even worse, it makes a lot of sense to address the problem at that time.